packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -0,0 +1,172 @@
+
+In both Identity Engine and Classic Engine, if the user has a session, the previously satisfied authenticators are considered before prompting for factors that are required by the `acr_values` parameter in the request.
+
+In both Identity Engine and Classic Engine, if the user is unable to satisfy the level of assurance, Okta returns an error to the callback endpoint.
+
+### Factor enrollment
+
+If the user hasn't enrolled a factor, during step-up authentication the user is prompted to enroll a factor as long as an [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy) is configured (or the [Multifactor Enrollment Policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy) in Classic Engine).
+
+## Predefined parameter values
+
+The following predefined optional parameters are available for use in your authorization requests:
+
+> **Note:** Any unsupported `acr_values` sent in the authorize request are ignored.
+
+| Parameter value<br> for `acr_values` | Description           | Parameter Type   | DataType   |
+| :----------------------- | :----------------------------------------- | :--------------- | :--------- |
+| `urn:okta:loa:1fa:pwd`   | Password only. Allows one factor authentication that requires the user’s password. | Query | String |
+| `urn:okta:loa:1fa:any`   | Any one factor. Allows one factor authentication with no requirements on which factor. | Query | String |
+| `urn:okta:loa:2fa:any`   | Any two factors. Allows two factor authentication with no requirements on which factors. | Query | String |
+| `phr`<br><br><ApiLifecycle access="ie" /> | Phishing-Resistant. Requires users to provide possession factors that cryptographically verify the sign-in server (the origin). Currently, only FIDO2/WebAuthn satisfies this requirement. Because phishing resistance implies [device binding](https://help.okta.com/okta_help.htm?type=oie&id=ext-configure-authenticators), that constraint is selected automatically when `phr` is specified. | Query | String |
+| `phrh`<br><br> <ApiLifecycle access="ie" /> | Phishing-Resistant Hardware-Protected. Requires that you store keys being used to authenticate in secure hardware (TPM, Secure Enclave) on the device. Currently, only Okta Verify meets this constraint. Because hardware protection implies [device binding](https://help.okta.com/okta_help.htm?type=oie&id=ext-configure-authenticators), that constraint is selected automatically when `phrh` is specified. | Query | String |
```

Comment:
We need to add one more acr to the table (we were not sure before but we are gonna make it in monthly release)

| | | | |
| --- | --- | --- | --- | 
| `urn:okta:loa:2fa:any:ifpossible` | Any two factors if possible. Allows two factor authentication with no requirements on which factors. Any two factors are prompted only if they are enrolled otherwise any one factor is prompted | Query | String |